packages/webcert/src/store/api/apiMiddleware.ts

```diff
@@ -28,8 +29,13 @@ const handleApiCallBegan: Middleware = ({ dispatch }: MiddlewareAPI) => () => as
       method,
       data,
       withCredentials: true,
+      headers: addLaunchIdIfPresent(),
     })
 
+    if (launchIdPresentInResponse(response) && !sessionStorage.getItem('launchId')) {
```

Comment:
I believe this code should belong to the userMiddleware. The launchId is part of the user and is not generic logic. If the user changes this logic need to change.